src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -194,6 +196,14 @@ public Optional<SecurityResponse> reRequestAuthentication(final SecurityRequest
         );
     }
 
+    @Override
+    public Set<String> getSensitiveUrlParams() {
+        if (jwtUrlParameter != null) {
+            return Set.of(jwtUrlParameter);
```

Comment:
are all jwt parameters considered sensitive?

Reply:
No, only URL params that would be considered a user's credentials. Normally, credentials are passed in the HTTP `Authorization` header, but the JWT authenticator in the security plugin also supports passing a JWT as a url param. Consider this request: `curl -XGET https://localhost:9200/_plugins/_security/authinfo?token=<jwt>&verbose=true`, the jwt is sensitive but the value of verbose is not. I have a test that verifies this behavior.